workshops/management/commands/upgrade_instructor_profiles.py

```diff
@@ -290,11 +290,12 @@ def check_entry(self, entry):
                                           entry['family']))
 
             # check if the person really is an instructor
-            instructor_badge = Badge.objects.get(name='instructor')
-            if person and instructor_badge not in person.badges.all():
+            instructor_badges = Badge.objects.filter(
```

Comment:
We can use a hard-coded list of badge names, like `INSTRUCTOR_BADGES = ['dc-instructor', 'swc-instructor']`.